src/go/k8s/main.go

```diff
@@ -179,8 +179,8 @@ func main() {
 			os.Exit(1)
 		}
 		hookServer := mgr.GetWebhookServer()
-		hookServer.Register("/mutate-redpanda-vectorized-io-v1alpha1-console", &webhook.Admission{Handler: &redpandawebhooks.ConsoleDefaulter{Client: mgr.GetClient()}})
-		hookServer.Register("/validate-redpanda-vectorized-io-v1alpha1-console", &webhook.Admission{Handler: &redpandawebhooks.ConsoleValidator{Client: mgr.GetClient()}})
+		hookServer.Register("/mutate-redpanda-vectorized-io-v1alpha1-console", &webhook.Admission{Handler: &redpandawebhooks.ConsoleDefaulter{Client: mgr.GetClient(), Logger: ctrl.Log.WithName("controllers").WithName("redpanda").WithName("Console")}})
```

Comment:
In the defaulter there isn't any logging done. Could we not populate logger?

Reply:
Naming nit: this isn't a controller, redpanda is redundant as we already know what log we're looking at. This should probably be just "Console.webhook".